calcom/cal.com
# fix: Allow lowercase/uppercase/mixedcase access  with org team event booking page

## What does this PR do?

Allow team event booking page for an org to be accessed in all cases.
Fixes #11719 

## Type of change

## How should this be tested?
- Create an org with slug org1 and create a team with slug `marketing` in it. 
- For that team, create an event `syncup`
- Now all of these are different ways to access the same event booking page
org1.cal.local:3000/Marketing/syncup and org1.cal.local:3000/MARketing/syncup and org1.cal.local:3000/marketing/syncup - All of these should work


## Mandatory Tasks
